certified-connectors/DocuSignDemo/script.csx

```diff
@@ -195,55 +209,828 @@ public class Script : ScriptBase
                 ["type"] = "object",
                 ["properties"] = new JObject
                 {
-                  ["documentId"] = new JObject
+                  ["groupName"] = new JObject
+                  {
+                    ["type"] = "string",
+                    ["x-ms-summary"] = "Group Name"
+                  },
+                  ["radios"] = new JObject
+                  {
+                    ["type"] = "array",
+                    ["items"] = new JObject
+                      {
+                        ["type"] = "object",
+                        ["x-ms-summary"] = "Radios",
+                        ["properties"] = new JObject
+                        {
+                          ["anchorString"] = new JObject
+                          {
+                            ["type"] = "string",
```

Comment:
I see bunch of code duplication. 
Can you DRY them out into a separate JObject and add only the changing ones for each.

Reply:
I thought about doing that. But there aren't tabs many tabs that have the exact same fields, so overlapping only the common fields looked more difficult to read since we would have to scroll up and down frequently to understand what fields are added in properties object.

Reply:
I see. Ok then.